packages/next/src/client/components/router-reducer/get-prefetch-cache-entry-status.ts

```diff
@@ -0,0 +1,37 @@
+import { PrefetchCacheEntry } from './router-reducer-types'
+
+const FIVE_MINUTES = 5 * 60 * 1000
+const THIRTY_SECONDS = 5 * 1000
```

Comment:
5 * 1000 milliseconds is 5 (five) seconds, not thirty (30) seconds

Reply:
good catch :) leftover from debugging